apps/admin/frontend/src/api.ts

```diff
@@ -208,7 +208,7 @@ export const getWriteInImage = {
   },
 } as const;
 
-type GetPrintedBallotsProps = QueryProps<'getPrintedBallots'>;
+type GetPrintedBallotsProps = QueryInput<'getPrintedBallots'>;
```

Comment:
Can you also change `GetPrintedBallotsProps` to `GetPrintedBallotsInput` and then change `queryKey(props)` to `queryKey(input)`? Sorry if I wasn't clear in my original suggestion

Reply:
You were clear, I didn't think it through.

Reply:
Fixed.